<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnrollNewQualityScreen</class>
 <widget class="QDialog" name="EnrollNewQualityScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enroll New Quality</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>531</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Current Quality : xxxxxxxxxxxxx</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>331</width>
     <height>221</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>329</width>
      <height>219</height>
     </rect>
    </property>
    <widget class="QListView" name="listViewForNewQuality">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>331</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="enrnqsenrollnewBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Enroll New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enrnqsbackBtn">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>590</y>
     <width>246</width>
     <height>59</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enrnqssavenewqualityBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>590</y>
     <width>221</width>
     <height>59</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Save New Quality</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
